docs/process/labels.md

```diff
@@ -3,40 +3,47 @@
-
-This section is organized into sub-groups, as groups of labels relate to
-specific work being done in the issue tracker.
-
-### Issue triage
+help identify what area a task is associated with. This helps organize upcoming
+work and enables contributors to focus on a specific area of the project that
+they are passionate about.
+
+|                                       | Label name               | Description |
+| ------------------------------------- | -------------------------| ----------- |
+| [:mag_right:][area:accessibility]     | `area:accessibility`     | Accessibility issues or improvements in the product |
+| [:mag_right:][area:dependencies]      | `area:dependencies`      | Upgrading dependencies that the project uses |
+| [:mag_right:][area:diffs]             | `area:diffs`             | Diff rendering and our use of [CodeMirror](https://codemirror.net/) |
+| [:mag_right:][area:docs]              | `area:docs`              | Documentation for the project |
+| [:mag_right:][area:electron]          | `area:electron`          | How we use [Electron](https://electronjs.org) in the product |
+| [:mag_right:][area:enhancements]      | `area:enhancements`      | Improvements to the product that address a specific problem for users |
+| [:mag_right:][area:external-websites] | `area:external-websites` | External websites that the GitHub maintainers are responsible for |
+| [:mag_right:][area:integrations]      | `area:integrations`      | Editor and shell integrations that ship in the product |
+| [:mag_right:][area:metrics]           | `area:metrics`           | Required to support analytics and metrics in the product |
+| [:mag_right:][area:performance]       | `area:performance`       | Improvements to the product to ensure it is responsive for users|
+| [:mag_right:][area:tech-debt]         | `area:tech-debt`         | Keep the application code maintainable so contributors can continue building features |
+| [:mag_right:][area:themes]            | `area:themes`            | The light or dark themes that ship in the product  |
+| [:mag_right:][area:tooling]           | `area:tooling`           | Scripts and tooling for the project |
+
+## Issue triage
 
 The triage process is how the maintainers process incoming issues and prioritize
 the work required to address the feedback raised. These labels help us to track
 the flow of an issue through this process
 
-|                                        | Label name                  | Description |
-| -------------------------------------- | ----------------------------| ----------- |
-| [:mag_right:][bug]                     | `bug`                     | Confirmed bugs or reports that are very likely to be bugs |
-| [:mag_right:][enhancement]             | `enhancement`             | Issues that propose to improve the app and solve a problem for users |
-| [:mag_right:][investigation-needed]    | `investigation-needed`    | Likely bugs, but haven't been reliably reproduced by a reviewer |
```

Comment:
I've removed `investigation-needed` in favour of these labels that I think better reflect the transition from issue triage to product work:

  - `triage:potential-bug` - the step in between confirming a bug or requiring more information
  - `triage:confirmed-bug` - the bug has been confirmed by a reviewer using the information provided
  - `triage:not-a-bug` - the triage process is done and the issue isn't related to a bug, and it's up to someone else to figure out the next steps (e.g. ask clarifying questions about the use case, understand the value, and either move it to `area:enhancement` if it could be done in Desktop, or close the issue to indicate that it's not in scope)